.github/workflows/pypi-publish.yml

```
@@ -17,16 +17,17 @@ jobs:
     # happen for this actual commit (the commit that the tag points to).
     # It also restores the files timestamps.
     - name: Download wheels from commit ${{ github.sha }}
-      uses: dawidd6/action-download-artifact@v2
+      uses: dawidd6/action-download-artifact@v8
       with:
         workflow: python-package.yml
         workflow_conclusion: success
         commit: ${{ github.sha }}
-        name: wheels
+        name_is_regexp: true
+        name: wheel-.*
```

```suggestion
        name: `wheel-.*`
```
Just in case the YAML parser decides to do funny things